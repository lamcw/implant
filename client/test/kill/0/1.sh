#!/bin/sh

./sc-client kill 2>&1 | sed 's/\x1b\[[0-9;]*m//g' > output.txt
diff -Z output.txt client/test/kill/0/1.stdout