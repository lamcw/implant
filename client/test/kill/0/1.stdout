[*] pid is a required argument.
infect
disinfect
exec [--bash] [--hide] [--uid <uid>] [--gid <gid>] <command>
kill <pid>
hide [--pid <pid> | --module]
unhide [--pid <pid> | --module]
set <pid> [--uid <uid>] [--gid <gid>]
unset <pid> [--uid] [--gid]
payload [--list | --exec <name>]
help <module>
